src/views/viewer/ChannelView/ChannelView.style.tsx

```diff
@@ -126,15 +133,82 @@ export const PaginationContainer = styled.div`
 `
 
 export const TabsContainer = styled.div`
+  display: grid;
   margin-bottom: ${sizes(8)};
-  border-bottom: solid 1px ${colors.gray[800]};
-
+  gap: ${sizes(2)};
+  grid-template-rows: 1fr 1fr;
+  grid-template-columns: 1fr 1fr;
+  grid-template-areas:
+    'tabs tabs tabs'
+    'search sort sort';
+  align-items: baseline;
   ${media.compact} {
     padding-top: ${sizes(8)};
   }
+  ${media.small} {
+    border-bottom: solid 1px ${colors.gray[800]};
+    grid-template-areas:
+      'tabs tabs '
+      'search  sort';
+  }
+  ${media.medium} {
+    grid-template-areas: initial;
+    gap: ${sizes(8)};
+    grid-template-rows: 1fr;
+    grid-template-columns: auto 1fr 250px;
+  }
```

Comment:
Not sure if it's in the scope of this PR, but I see in the design hover state for Tabs
![image](https://user-images.githubusercontent.com/51168865/126761205-1969b75b-d024-465c-871c-41c4aa9d1abf.png)

Reply:
Good catch. Doesn't have to be in this PR but let's create a followup issue to tackle that @eldiegod

Reply:
One thing I'd like to request here, I think the Information tab should have it's own URL so you can link directly to channel details. Maybe something like `/channel/1` / `/channel/1?tab=videos` for videos and `/channel/1?tab=info` for information?

Again, probably not in scope for this PR but let's create an issue for that as well

Reply:
Made a new issue for the hover states